bin/rpc/rpc.ml

```diff
@@ -1,5 +1,5 @@
 module Group = Group
-module Rpc_build = Rpc_build
-module Rpc_common = Rpc_common
-module Rpc_ping = Rpc_ping
-module Rpc_status = Rpc_status
+module Build = Rpc_build
+module Common = Rpc_common
+module Ping = Rpc_ping
+module Status = Rpc_status
```

Comment:
If you'd like to rename these modules, you should rename their source names as well. Renaming just how they're exported makes things needlessly inconsistent.